README.md

```diff
@@ -12,84 +12,41 @@ Privacy Badger is a project of the Electronic Frontier Foundation.
 
 ## Developing
 
-### Load the extension from source code
+We're glad you want to help! We have a **[handy guide for
+contributors](/doc/CONTRIBUTING.md)**. It can hopefully point you to helpful
+resources if you have any questions (please let us know if it doesn't). To get
+started with most things, you'll need to [install Privacy Badger from
+source](#install-from-source)  But to hack on Privacy Badger you need to install
+it from source first, so we'll start there.
```

Comment:
Seems repetitive ("need to install from source, need to install from source first"), not sure what you mean.